src/SFA.DAS.Forecasting.Database/Tables/Payment.sql

```diff
@@ -3,7 +3,7 @@
 	[Id] BIGINT NOT NULL PRIMARY KEY Identity(1,1),
 	[ExternalPaymentId] NVARCHAR (100) NOT NULL,
 	[EmployerAccountId] BIGINT NOT NULL,
-	[SendingEmployerAccountId] BIGINT NOT NULL default(0),
+	[SendingEmployerAccountId] BIGINT NOT NULL,
```

Comment:
Why has the default constraint been removed? Can you add the default and name it please e.g.
[SendingEmployerAccountId] BIGINT NOT NULL Constraint DF_Commitment__SendingEmployerAccountId default(0),